moleculemaker/chemscraper-frontend
# Update domain for production ChemScraper frontend

## Problem
We would like a nicer URL than `chemscraper.frontend.mmli1.ncsa.illinois.edu`

While helpful for development, this pattern is too long and complex for typical users to remember

## Approach
* fix: adjust prod domain to `chemscraper.platform.moleculemaker.org`
* feat: add a redirect from existing domain to new one (to prevent existing links from breaking)

## How to Test
1. Navigate to https://chemscraper.frontend.mmli1.ncsa.illinois.edu
    * You should be automatically redirected to the new domain: https://chemscraper.platform.moleculemaker.org
    * You should see that any path you include (`/configuration`, `/results/:id`, etc) are preserved
